### DISCUSSION TITLE
Weekly sync  2 May 2022

### DISCUSSION THREAD

**Discussion by sabineschaller** [interledger/rafiki#330] [General]
## Updates 

- Brandon
  - opened quotes PR - under review
  - started working on OPv2 OpenAPI spec for json validation in http requests
  - found issues in the spec itself
  - koa middleware don't support v 3.1, which we are on, just yet
  - hopefully, we can replace custom solution in the future

- Sabine
  - still working on [list endpoint PR](https://github.com/interledger/rafiki/pull/323)
    - abstracting logic
    - working on edge cases like underlying TB accounts not found
  - working on WM via OP flows to figure out how to make WM as privacy-preserving as possible
    - possible change to spec: 
      - not include account details in incoming payment URL
      - not include account-`identifier` in incoming payment GET requests

- Nathan
  - still working on auth service to identify clients by key
  - uses that to establish testing
  - moving to grant flow

- Matt
  - Don peered 2 rafiki instances
  - ilp-pay does not support GNAP tokens
  - Don to create a list of all the issues
  - payment was successfully sent!!!

## Timelines & Project Scope
[ Main milestones](https://github.com/interledger/rafiki/milestones)

## Review/Spec/TODO
- [quotes PR](https://github.com/interledger/rafiki/pull/326) @sabineschaller @matdehaast @DonChangfoot @njlie 
- review ILF typescript developer role challenge submissions @sabineschaller @njlie

**Comment by A-Mozeak** [comment]
Hello! We tried to join the weekly call at 1700 GMT (1pm ET for us) but it looks like the call was an hour earlier. Is the meeting taking place at 1600 GMT going forward?

Also, is the Docker image for Rafiki available for public access?

**Reply by wilsonianb** [reply]
Sorry about that. The meeting is currently 1600 GMT. The Google Calendar event in the README appears to have the correct time:
https://github.com/interledger/rafiki#planning-calls

The Docker image is available on GitHub: https://github.com/interledger/rafiki/pkgs/container/rafiki-backend
However, it is slightly out date due to this open issue: https://github.com/interledger/rafiki/issues/321

**Reply by A-Mozeak** [reply]
Noted, thanks!
